Rakefile

```diff
@@ -1,8 +1,6 @@
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 
-RSpec::Core::RakeTask.new(:spec) do |task|
-  task.ruby_opts = "-W"
```

Comment:
@maxjacobson is probably gonna nit you for this 😉. The warning flag previously helped find some pretty old Ruby constructs. Maybe we don't need it anymore though it there are no current warnings? (We're probably unlikely to introduce new ones.)